themes/src/main/resources/theme/base/admin/resources/partials/realm-identity-provider-oidc.html

```diff
@@ -138,6 +138,20 @@
                 </div>
                 <kc-tooltip>{{:: 'token-url.tooltip' | translate}}</kc-tooltip>
             </div>
+            <div class="form-group clearfix">
```

Comment:
Switch was already added and in the other PR, you already uncommented the line allowing clientAuth_basic in the existing switch. That should be sufficient to uncomment that line. No need to add the switch again to have it duplicated on the page. Could you please remove the duplicated switch from your PR?